src/extension_instance.ml

```diff
@@ -3,6 +3,8 @@ open Import
 type t =
   { mutable sandbox : Sandbox.t
   ; mutable repl : Terminal_sandbox.t option
+  ; mutable ocaml : Ocaml.t option
+        (** invariant: must be set before initializing the language server *)
```

Comment:
Do we check this somewhere in the code?

Reply:
We access that field with `Option.value_exn`, which is kind of a check. Do you have a preferred place you'd like it checked?

Reply:
I see. I suppose that's fine then.